examples/bloom_filter/host_bulk_example.cu

```diff
@@ -25,15 +25,21 @@
 
 int main(void)
 {
-  // Generate 10'000 keys and insert the first 5'000 into the filter.
-  int constexpr num_keys = 10'000;
-  int constexpr num_tp   = num_keys * 0.5;
-  int constexpr num_tn   = num_keys - num_tp;
+  int constexpr num_keys    = 10'000;          ///< Generate 10'000 keys
+  int constexpr num_tp      = num_keys * 0.5;  ///< Insert the first half keys into the filter.
+  int constexpr num_tn      = num_keys - num_tp;
+  int constexpr sub_filters = 200;  ///< 200 sub-filters per bloom filter
```

Comment:
Wait, why did clang-format re-format all this stuff? Did we recently (between me adding this example a month ago and now) change the clang-format version or something? @PointKernel

Reply:
no, nothing changed.

Reply:
@mhaseeb123 are you using the pre-commit hook for code formatting? https://github.com/NVIDIA/cuCollections?tab=readme-ov-file#pre-commit-hook

Reply:
Yes, I am!

Reply:
I think clang-format did this as I added the `int constexpr sub_filters = 200` line which was a magic number before. Since it's declared with other variables, clang-format tries to add spaces such that the `=` is aligned for all variables in the list

Reply:
Whups, my bad, I missed that part.